newsfragments/1774.feature

```diff
@@ -0,0 +1 @@
+``dials.image_viewer``: adjust brightness control for better defaults.
```

Comment:
Better is an opinion, would say they are certainly better for photon counting detectors